sdk/identity/Azure.Identity/src/Azure.Identity.csproj

```
@@ -16,13 +16,12 @@
       <PropertyGroup>
         <DefineConstants>PREVIEW_FEATURE_FLAG</DefineConstants>
       </PropertyGroup>
-      <ItemGroup>
-        <PackageReference Include="Azure.Core.Experimental" />
-      </ItemGroup>
     </When>
   </Choose>
   <ItemGroup>
-    <PackageReference Include="Azure.Core" />
+    <!-- TODO: Revert after Azure.Core ships
+      <PackageReference Include="Azure.Core" /> -->
+    <projectreference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

```suggestion
    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

XML is case-sensitive normally, isn't it?